extensions/wikia/PortableInfoboxBuilder/PortableInfoboxBuilderHooks.class.php

```diff
@@ -22,4 +22,98 @@ public static function onSkinAfterBottomScripts( $skin, &$text ) {
 
 		return true;
 	}
+
+	/**
+	 * Hook that exports url to the current template page
+	 *
+	 * @param Array $vars - (reference) js variables
+	 * @param Array $scripts - (reference) js scripts
+	 * @param Skin $skin - skins
+	 * @return Boolean True - to continue hooks execution
+	 */
+	public static function onWikiaSkinTopScripts( &$vars, &$scripts, $skin ) {
+		$title = $skin->getTitle();
+
+		if ( $title && $title->isSpecial( PortableInfoboxBuilderSpecialController::PAGE_NAME ) ) {
+			// remove the special page name from the title and return url to the template
+			// passed after the slash, i.e.
+			// Special:InfoboxBuilder/TemplateName/Subpage => Template:TemplateName/Subpage
+			$vars['templatePageUrl'] = Title::newFromText(
+				implode( '/', array_slice( explode( '/', $title->getText() ), 1 ) ),
+				NS_TEMPLATE
+			)->getFullUrl();
+		}
```

Comment:
Added in https://github.com/Wikia/app/commit/d93c27d513861d24c3d54c0b6e8d82796054c480